graf3d/g3d/src/THelix.cxx

```diff
@@ -38,12 +38,12 @@ The second constructor has 6 parameters,
 Example:
 
 ~~~ {.cpp}
-      THelix pl1(xyz, v, w, range, rtype, axis);
+      THelix pl1(p, v, w, range, rtype, axis);
 ~~~
 
 where:
 
-  - xyz  : array of initial position
```

Comment:
I think`xyz` conveys that it's 3-dimensional, something `p` does not. Maybe consider keeping `xyz`? Even better might be to change `Double_t const* p` to `Double_t p[3]`!

Reply:
yes I will go for xyz. But it needs to be consistent between the .h and the .cxx (which was not)